internal/incident/incident.go

```diff
@@ -381,33 +382,35 @@ func (i *Incident) evaluateEscalations(ctx context.Context, tx *sqlx.Tx, ev *eve
 	return nil
 }
 
-// notifyContacts executes all the given pending notifications of the current incident.
+// notifyContacts executes all the given pending notifications of the current incident in a go routine.
```

Comment:
With that documentation, it reads like it would be done in the background, but this function waits for it. I'd just leave the documentation here as-is as the change is not relevant for the caller.